pyxdf/pyxdf.py

```diff
@@ -94,6 +95,19 @@ def load_xdf(filename,
     Args:
         filename : name of the file to import (*.xdf or *.xdfz)
 
+        select_streams : One or more stream IDs to load. This can be an integer
+          integer, a list of integers or a list of dicts. For example,
```

Comment:
duplicated "integer". My proposal:

```
select_streams: int | list[int] | list[dict]
  One or more stream IDs to load. The accepted values are:
  `int` / `list[int]`: load only specified stream ids, e.g.
  `[2,4]` loads only …
  `list[dict]`: only load streams matching the query, e.g.
   …
```

That way, it's easier to grok what values are allowed.

Reply:
OK, I've rephrased the text (and also included text for the value `None`).